apps/examples/testcase/ta_tc/systemio/itc/itc_i2c.c

```diff
@@ -115,13 +115,20 @@ void itc_iotbus_i2c_set_address_p(void)
 void itc_iotbus_i2c_write_read_p(void)
 {
 	int ret = IOTBUS_ERROR_NONE;
-	uint8_t szCmd[2] = { 0x01, 0x02 };
-	uint8_t szReadBuf[10];
+	uint8_t szCmd[1] = { 0x01};
+	uint8_t address = 0x23;
+	uint8_t szReadBuf[1];
```

Comment:
Here is a Tizen Coding rule. Please check it.
[M12] No mixed-case, Use underscores('_') to separate words in an name.

[Example-Do]
```
#define TEST_VALUE	32
extern struct list_head_hash[TEST_VALUE];

int example_function(void)
{
    return(TEST_VALUE - 1);
}
```

[Example-Don't]
```
#define Test_Value	32
extern struct list_head_hash[Test_Value];

int ExampleFunction(void)
{
    return(Test_Value - 1);
}
```